tap_helpshift/client.py

```diff
@@ -162,8 +164,8 @@ async def _get_resp_text(resp):
                 wait_s = 0
 
                 try:
-                    LOGGER.info('GET %s %r', url, params)
-                    async with self.session.get(url, params=params) as resp:
+                    LOGGER.info('GET %s %r timeout=%s', url, params, timeout)
+                    async with self.session.get(url, params=params, timeout=timeout) as resp:
```

Comment:
Since it seems like all requests come through this method, why not do `timeout=DEFAULT_TIMEOUT` here? Then the other methods have no need to pass it through.